utilities_common/intf_filter.py

```diff
@@ -0,0 +1,43 @@
+# Interface filtering functions
+
+SONIC_PORT_NAME_PREFIX = "Ethernet"
+SONIC_PO_NAME_PREFIX = "PortChannel"
```

Comment:
Suggest to use LAG_NAME_PREFIX to be consistent with [swss-common](https://github.com/Azure/sonic-swss-common/blob/master/common/schema.h#L85)

Reply:
Could not find LAG_NAME_PREFIX.

Reply:
SONIC_PO_NAME_PREFIX -> SONIC_LAG_NAME_PREFIX - I put the link only as a reference